recipes/jupysql/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "jupysql" %}
+{% set version = "0.6.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupysql-{{ version }}.tar.gz
+  sha256: 8c328e18ff43f566905aaa8a26c7c4693d83be5ba5e69bd2bb6c1d6a3c3af865
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```

This package is pure Python and can be noarch:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  noarch: python
```